### ISSUE TITLE
Brave browser is blocking font in tiny-mce

### ISSUE THREAD

**Issue by Abhishek7709** [tinymce/tinymce#9309]
## 📝 Provide detailed reproduction steps (if any)
Please provide the steps to reproduce and if possible a minimal demo of the problem via [fiddle.tiny.cloud](fiddle.tiny.cloud) or similar.
1. …Brave browser is blocking font in tiny-mce.
2. …
3. …

### ✔️ Expected result
It should look font as same as paste.

### ❌ Actual result
*What is the actual result of the above steps?*

## 📃 Other details
- Browser: …Brave
- OS: …Windows
- First affected version: …
- Worked in version: …

---

If you'd like to see this fixed sooner, add a 👍 reaction to this post.

**Comment by spocke** [comment]
We need a bit more information here what font is being blocked and what are you trying to paste? So ideally we would need the HTML that is being pasted so that we can try to reproduce this issue.

**Comment by Abhishek7709** [comment]
freestyle script font is blocked on brave and safari  browser

**Comment by spocke** [comment]
TinyMCE will not be able to carry over custom fonts from a remote site into the editor unless the editor also has that custom font. Here is a blog article on how to add Google fonts into the editor: https://www.tiny.cloud/blog/tinymce-google-fonts/ it needs to be registered in the editor to work properly.

What is the HTML code that you get inside the editor if that still contains the free style font but it's not rendered inside the editor then it's likely that you are missing the custom font registration part.

If the font style is stripped from the HTML then there is something going on in our paste processing code and then we need to have the HTML before it's being pasted to be able to reproduce the issue and figure out what the problem might be.

**Comment by Abhishek7709** [comment]
Font is visible in editor list but if we paste email signature from outlook to editor, it is not visible as it is, but this is working in chrome.
if we make shield down, it is working fine.

**Comment by lostkeys** [comment]
Searching the web, this appears to happen because fonts can be used to fingerprint and track you, and brave blocks the fonts. This is not something we can address in TinyMCE. I will go ahead and close this issue, but if you think this issue is not caused by Brave, please submit a new issue with more details, perhaps a screen recoriding and a replication case that we can investigate with. Thanks

— Fredrik, Product Manager
